.github/workflows/macos_build_ios.yml

```diff
@@ -147,7 +148,7 @@ jobs:
       env:
         BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
       run: |
-        curl -u "planetit1:$BROWSERSTACK_ACCESS_KEY" -X POST "https://api-cloud.browserstack.com/app-live/upload" -F "file=@uploads/$upload_file"
```

Comment:
Don't we want to be able to test the Treemapper app anymore with BrowserStack App Live?

Reply:
We can still upload the builds from drive. 

App automate can run automation based on the tests we write. Locally or on browser stack web.
I would say this is an upgrade.

Reply:
Aren't we allowed to upload the files to both services at BrowserStack using two curl commands?

Reply:
We can definitely do that.